# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/OriginOperations.h
# Commit: b12a2f55233f
# Full Hash: b12a2f55233fade8f2e3411295ff3d362df9e359
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2024-10-09 21:46:09
# Regressor Bug: 1866402
# File Overlap Count: 1
# Description:
#   Bug 1866402 - Change nsIQuotaManagerService::ResetStoragesForPrincipal to support resetting of entire origins only; r=dom-storage-reviewers,jari
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195510
# ==============================================================================

diff -r 6a90f36c8bf5 -r b12a2f55233f dom/quota/OriginOperations.h
--- a/dom/quota/OriginOperations.h	Wed Oct 09 11:59:34 2024 +0000
+++ b/dom/quota/OriginOperations.h	Wed Oct 09 11:59:35 2024 +0000
@@ -136,8 +136,7 @@
 RefPtr<ResolvableNormalOriginOp<bool>> CreateShutdownOriginOp(
     MovingNotNull<RefPtr<QuotaManager>> aQuotaManager,
     Maybe<PersistenceType> aPersistenceType,
-    const mozilla::ipc::PrincipalInfo& aPrincipalInfo,
-    Maybe<Client::Type> aClientType);
+    const mozilla::ipc::PrincipalInfo& aPrincipalInfo);
 
 RefPtr<ResolvableNormalOriginOp<bool>> CreateShutdownClientOp(
     MovingNotNull<RefPtr<QuotaManager>> aQuotaManager,